<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetsManager</class>
 <widget class="QWidget" name="WidgetsManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>113</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WidgetsManager</string>
  </property>
  <layout class="QVBoxLayout" name="centralLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,1,0,0,3,0">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="palyButton">
        <property name="toolTip">
         <string>Play All</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/png-icons/design-collection/play-1.png</normaloff>:/png-icons/design-collection/play-1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pauseButton">
        <property name="toolTip">
         <string>Pause All</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/png-icons/design-collection/pause-1.png</normaloff>:/png-icons/design-collection/pause-1.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="recordButton">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources/icons.qrc">
          <normaloff>:/png-icons/design-collection/rec-2.png</normaloff>
          <normalon>:/png-icons/media-collection/stop.png</normalon>:/png-icons/design-collection/rec-2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="saveToLabel">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save To:  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="saveToButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset>
          <disabledoff>:/png-icons/design-collection/folder.png</disabledoff>
         </iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="saveToDirLineEdit">
        <property name="text">
         <string>/home/javad/workspace/test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="saveToFileLabel">
        <property name="text">
         <string>/log.json</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>479</width>
        <height>68</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
